### DISCUSSION TITLE
Add the ability to recycle a post

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-future#372] [Feature Requests]
This could be a "reset" feature to keep old posts fresh:

> Would it be possible to re-post in the future? We have a post type that "recycles".... Each day (at a set time) the oldest published post in the post type is unpublished, cleared of comments and republished as new.

It's worth thinking through how it could work in terms of the settings and options? Applying this to one post would be simple. Applying to a post type and sorting by age would be trickier.

**Comment by jeseph** [comment]
Thanks Steve!

Our use-case is certainly specific but I could see some value for others in being able to have a "looping" set of posts. Our example can be seen here: https://gratefulness.org/practice/daily-questions/

**Comment by stevejburge** [comment]
Thanks @jeseph. Maybe there could be an option to choose specific posts. You could manually order those posts and then set them to loop on a schedule.

**Comment by jeseph** [comment]
Would likely work for many, we have ~430 posts 😅

**Comment by stevejburge** [comment]
OK, thanks @jeseph. This sounds like the kind of feature where might need to start small and build. I can imagine starting with the ability to recycle individual posts for example, and then adding from there.
